### ISSUE TITLE
Add Bootstrap back in

### ISSUE THREAD

**Issue by kylemacey** [kylemacey/crystal-tiger-dice#1]

**Comment by bryantheastronaut** [comment]
Are you planning on using a module like [react-bootstrap](http://react-bootstrap.github.io/) or the regular one via a link tag in the _index.html_ header?

**Comment by kylemacey** [comment]
I thought about checking out react-bootstrap. I might go that route, it seems neat. Though, I was originally thinking of having the header/footer in `public/index.html` and have them be static. I'm not sure how well that would work.

**Comment by bryantheastronaut** [comment]
Yeah I'm not really sure about that. I've never mixed it like that before. Seems like it may be an anti-pattern to me but thats just my gut. You could also just put them in the _App.js_ component sandwiched around the `<Game />`.
